README.md

```diff
@@ -59,6 +59,55 @@ group_delta = {"variables"=>{"key"=>"value"}, "id"=>my_group_id, "classes"=>{"mo
 puppetclassify.groups.update_group(group_delta)
 ```
 
+### Retrieving classification of a node
+
+Because the Console classifies nodes based on rules, you may want to submit a
+complete `facts` hash for rules to match. See [the API docs](https://docs.puppetlabs.com/pe/latest/nc_classification.html)
+for examples.
+
+You can retrieve facts for the local node either using Facter from Ruby or the
+command line. For example:
+
+```ruby
+#! /opt/puppetlabs/puppet/bin/ruby
+require 'facter'
+
+facts = { 'fact' => Facter.to_hash }
+```
+
+or:
+
+```ruby
+#! /usr/bin/env ruby
+require 'json'
+
+facts = { 'fact' => JSON.parse(`facter -j`) }
+```
+
+Once you have the facts, retrieving classification of a node is simple:
+
+```ruby
+#! /opt/puppetlabs/puppet/bin/ruby
+require 'facter'
```

Comment:
Yeah, you'll have to set the GEMPATH or just use /opt/puppetlabs/puppet/bin/ruby